apps/console/src/init/worker.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const ctx: Worker = self as any;
+
+setInterval(function() {
+    ctx.postMessage('');
+}, 1000);
+
```

Comment:
Don't we need to add this (`export default {} as typeof Worker & { new (): Worker };`) here?

Reply:
Removed the file as the worker is moved to the core module.